src/sentry/loader/browsersdkversion.py

```diff
@@ -4,11 +4,10 @@
 import re
 
 from django.conf import settings
-from pkg_resources import parse_version
+from packaging.version import Version
```

Comment:
`packaging.version.parse` falls back to `LegacyVersion`, however we ensure up front that the versions are of the form `\d+\.\d+\.\d+` so they fit the strict version parsing